src/Vehicle/Vehicle.cc

```diff
@@ -873,6 +874,14 @@ void Vehicle::_handleStatusText(mavlink_message_t& message)
     strncpy(b.data(), statustext.text, MAVLINK_MSG_STATUSTEXT_FIELD_TEXT_LEN);
     b[b.length()-1] = '\0';
     messageText = QString(b);
+    droneNumber = statustext.text;
+    if(droneNumber.contains("PX4v") || droneNumber.contains("Pixhawk") || droneNumber.contains("fmuv") || droneNumber.contains("CubeOrange")){
```

Comment:
What will it take to support more FC's? Should this be coming from some vector of strings we maintain elsewhere in some header?